src/components/booksComponent/BooksItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const BooksItem = () => (
+  <div>
+    <h2>Add New Book</h2>
+    <form className="form">
+      <input type="text" placeholder="Book Title" />
+      <input type="text" placeholder="Categorie" />
+      <button type="submit">Add Book</button>
+    </form>
+  </div>
+);
```

Comment:
- [ ] (Optional) It would be nice to transfer this code to your `AddBook` component for semantic purposes because the actual function of the code is to add a book after the button is clicked.